applications/CompressiblePotentialFlowApplication/custom_processes/apply_far_field_process.cpp

```diff
@@ -0,0 +1,128 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Inigo Lopez and Marc Nunez
+//
+
+// Project includes
+#include "apply_far_field_process.h"
+
+namespace Kratos {
+
+// Constructor for ApplyFarFieldProcess Process
+ApplyFarFieldProcess::ApplyFarFieldProcess(ModelPart& rModelPart, const double rReferencePotential)
+    : Process(), mrModelPart(rModelPart), mReferencePotential(rReferencePotential)
+{
+    mFreeStreamVelocity = mrModelPart.GetProcessInfo()[FREE_STREAM_VELOCITY];
```

Comment:
You can make this member variable `const` if you initialize it in the constructor list two lines above.